I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/script.rpy", line 122, in script
    mc "Tidak ada yang bisa dibicarakan lagi, wozky Semuanya terdengar dengan jelas, terimakasih atas kabarnya. Aku ingin bergegas ke rumah untuk bertemu dengan adikku [berjalan menuju jembatan]"
KeyError: u'berjalan menuju jembatan'

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/script.rpy", line 122, in script
    mc "Tidak ada yang bisa dibicarakan lagi, wozky Semuanya terdengar dengan jelas, terimakasih atas kabarnya. Aku ingin bergegas ke rumah untuk bertemu dengan adikku [berjalan menuju jembatan]"
  File "renpy/ast.py", line 706, in execute
    renpy.exports.say(who, what, *args, **kwargs)
  File "renpy/exports.py", line 1384, in say
    who(what, *args, **kwargs)
  File "renpy/character.py", line 1164, in __call__
    what = what_pattern.replace("[what]", sub(what, translate=True))
  File "renpy/character.py", line 1153, in sub
    return renpy.substitutions.substitute(s, scope=scope, force=force, translate=translate)[0]
  File "renpy/substitutions.py", line 270, in substitute
    s = formatter.vformat(s, (), kwargs)
  File "/home/tom/ab/renpy-build/tmp/install.android-x86_64/lib/python2.7/string.py", line 563, in vformat
  File "/home/tom/ab/renpy-build/tmp/install.android-x86_64/lib/python2.7/string.py", line 585, in _vformat
  File "/home/tom/ab/renpy-build/tmp/install.android-x86_64/lib/python2.7/string.py", line 646, in get_field
  File "/home/tom/ab/renpy-build/tmp/install.android-x86_64/lib/python2.7/string.py", line 605, in get_value
KeyError: u'berjalan menuju jembatan'

